packages/alfa-dom/src/node/attribute.ts

```diff
@@ -46,7 +48,15 @@ export class Attribute extends Node {
   }
 
   public get name(): string {
-    return this._name;
+    // HTML attributes are case insensitive. Other, e.g. SVG, are case sensitive.
+    return this._namespace.some(equals(Namespace.HTML)) ? this._name.toLowerCase() : this._name;
+  }
+
+  public matchName(name: string): boolean {
```

Comment:
```suggestion
  public hasName(name: string): boolean {
```

This should also be placed after the getters. We should enforce the ordering using https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/member-ordering.md once ESLint is up and running (#171).